app/code/community/TIG/MyParcel2014/Helper/Data.php

```diff
@@ -242,16 +242,18 @@ public function shippingMethodIsPakjegemak($method)
      */
     public function getCurrentOptionsHtml($myParcelShipment)
     {
+        $addressValidation = new TIG_MyParcel2014_Helper_AddressValidation;
         $options = array(
             $this->__(ucfirst(str_replace('_', ' ', $myParcelShipment->getShipmentType()))),
         );
 
         if ($myParcelShipment->getShipmentType() == 'normal') {
 
-            if ($myParcelShipment->getHomeAddressOnly() == '1')
+            $storeId = $myParcelShipment->getOrder()->getStoreId();
+            if ( $myParcelShipment->getHomeAddressOnly() == '1' && $addressValidation->hasAgeCheck( $storeId ) == false)
```

Comment:
```suggestion
            if ($myParcelShipment->getHomeAddressOnly() == '1' && $addressValidation->hasAgeCheck($storeId) == false)
```
Dit komt van jouw wp phpstorm instellingen, maar is niet een standaard. Dit moet je even aanpassen in al jouw code.

Reply:
Done